components/import-action/tab-content/tab-content.tsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import classNames from "classnames";
+import { CopyBox } from "@teambit/documenter.ui.copy-box";
+
+import styles from "./tab-content.module.scss";
+
+export type TabContentProps = {
+  titleLine: React.ReactNode;
+  copyString: string;
+  children: React.ReactNode;
+} & React.HTMLAttributes<HTMLDivElement>;
+
+export function TabContent({ titleLine, copyString, children }: TabContentProps) {
+  return (
+    <div>
+      <div className={styles.top}>
+        <div className={styles.text}>{titleLine}</div>
```

Comment:
how about `styles.text` -> `styles.title`, `titleLine` -> `title` ?

Reply:
it was title but it doesnt allow to pass non string props to it, so I changed to titleLine

Reply:
changed `styles.text` to `styles.title`